lang/smaily-for-wp-et.po

```diff
@@ -1,22 +1,21 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: smaily\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-10-05 15:41+0300\n"
-"PO-Revision-Date: 2020-10-05 15:41+0300\n"
-"Last-Translator: Smaily <info@smaily.com>\n"
-"Language-Team: Eesti\n"
+"Project-Id-Version: Smaily for WP\n"
+"POT-Creation-Date: 2021-01-06 12:29+0200\n"
+"PO-Revision-Date: 2021-01-06 12:30+0200\n"
+"Last-Translator: tomabel <tom@smaily.com>\n"
+"Language-Team: Sendsmaily OÜ <info@smaily.com>\n"
 "Language: et\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.3\n"
+"X-Generator: Poedit 2.4.1\n"
 "X-Poedit-Basepath: ..\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Poedit-KeywordsList: __;_e;esc_html__;esc_attr__;esc_attr_e;esc_html_e\n"
-"X-Loco-Version: 2.2.2; wp-5.1.1\n"
+"X-Poedit-KeywordsList: __;_e;__ngettext:1,2;_n:1,2;__ngettext_noop:1,2;"
+"_n_noop:1,2;_c;_nc:4c,1,2;_x:1,2c;_nx:4c,1,2;_nx_noop:4c,1,2;_ex:1,2c;"
+"esc_attr__;esc_attr_e;esc_attr_x:1,2c;esc_html__;esc_html_e;esc_html_x:1,2c\n"
```

Comment:
You can safely remove `__ngettext`, `__ngettext_noop`, `_c` and `_nc` functions from the keywords list, because these have been deprecated by WordPress.

Reply:
Removed in 79b8213